openfisca_core/tools/test_runner.py

```diff
@@ -165,15 +166,15 @@ def runtest(self):
         finally:
             tracer = self.simulation.tracer
             if verbose:
```

Comment:
Here depends on `average` and `max-depth` compatibility, a proposal:

```suggestion
            if verbose and aggregate:
                self.print_computation_log(tracer, aggregate = True)

            if verbose and max_depth:
                self.print_computation_log(tracer, max_depth = max_depth)

            if verbose:
                self.print_computation_log(tracer)
```